e3nn/o3/tensor_product/tensor_product.py

```diff
@@ -538,15 +523,12 @@ class FullyConnectedTensorProduct(TensorProduct):
         see `TensorProduct`
     """
     def __init__(
-            self,
-            irreps_in1,
-            irreps_in2,
-            irreps_out,
-            normalization='component',
-            internal_weights=None,
-            shared_weights=None,
-            _specialized_code=True,
-                ):
+        self,
+        irreps_in1,
+        irreps_in2,
+        irreps_out,
+        **kwargs
```

Comment:
The problem with `**kwargs`` is that it breaks autocompletion

Reply:
Ture, I forgot — feel free to change this back and just add an explicit `_optimize_einsums` param